README.md

```diff
@@ -3718,7 +3725,10 @@ resource cleanup when possible.
   ```
 
 * <a name="consistent-string-literals"></a>
-  Prefer single quotes for string literals.
+  Prefer double quotes when
+    * writing natural language (since it might contain apostrophes)
```

Comment:
Is this not just "When a string contains apostrophes"? I believe this kind of rule will be very hard to make work in rubocop - currently it alerts if double quotes are not required, but if we move to a "use for natural language" rule, it will alert on those unnecessarily, and also makes it more obscure when it is actually allowed.

Reply:
Well, having rubocop check this rule isn't terribly helpful anyway. Nothing bad happens when you use the "wrong" quotes.

Reply:
>The thing is that when the string contains text, it may be changed to need some interpolation in the future. It's weird to have to change the quotes when you go to add interpolation.

I don't find it all that weird; in fact I installed an [atom plugin to make switching quotes easier](https://atom.io/packages/toggle-quotes).

>As for rubocop, having it check this rule isn't terribly helpful since nothing bad happens when you use the "wrong" quotes.

Rubocop is about ensuring consistency more than anything else - nothing bad happens in regards to a lot of rules that can be broken. I suppose we could agree to the change here and turn off this rubocop rule, but it will make it harder - I would say impossible - to remain consistent on this style, because quotes are very easy to miss, and we will not have anything automated to check it.